neuro_san/internals/messages/chat_message_type.py

```diff
@@ -67,6 +67,9 @@ def from_response_type(cls, response_type: Union[str, ChatMessageType]) -> ChatM
         if isinstance(response_type, ChatMessageType):
             return response_type
 
+        if isinstance(response_type, int):
+            return ChatMessageType(response_type)
+
```

Comment:
Now that we lose gRPC response structure conversions,
we may get integer value for ChatMessageType, so construct its value from int.